tests/foundation/ethereum/foundation_ethereum_assert_trades.sql

```diff
@@ -0,0 +1,54 @@
+-- Check if all Ethereum Foundation trade events make it into the nft.trades
+WITH raw_events AS (
+  SELECT f.evt_block_time AS raw_block_time
+  , f.evt_tx_hash AS raw_tx_hash
+  , c.nftContract AS raw_nft_contract_address
+  , c.tokenId AS raw_token_id
+  FROM {{ source('foundation_ethereum','market_evt_ReserveAuctionFinalized') }} f
+  LEFT JOIN {{ source('foundation_ethereum','market_evt_ReserveAuctionCreated') }} c ON c.auctionId=f.auctionId AND c.evt_block_time<=f.evt_block_time
+  WHERE f.evt_block_time >= --'2022-04-15'
+  AND f.evt_block_time < NOW() - interval '1 day' -- allow some head desync
```

Comment:
This is the error in the test. Remove the commented date here...

PR with syntax error fix #1589